Library/Homebrew/completions/fish.erb

```diff
@@ -104,22 +104,14 @@ function __fish_brew_ruby_parse_json -a file parser -d 'Parses given JSON file w
 end
 
 function __fish_brew_suggest_formulae_all -d 'Lists all available formulae with their descriptions'
-    # store the brew cache path in a var (because calling (brew --cache) is slow)
-    set -q __brew_cache_path
-    or set -gx __brew_cache_path (brew --cache)
-
-    if test -f "$__brew_cache_path/descriptions.json"
-        __fish_brew_ruby_parse_json "$__brew_cache_path/descriptions.json" \
-            '.each{ |k, v| puts([k, v].reject(&:nil?).join("\t")) }'
-    else
-        brew formulae
-    end
+    brew formulae
 end
 
 function __fish_brew_suggest_formulae_installed
-    brew list --formula
+    command ls -1 (brew --cellar)
```

Comment:
```fish
❯ hyperfine 'brew list --formula' 'command ls -1 (brew --cellar)' --shell fish
Benchmark 1: brew list --formula
  Time (mean ± σ):     473.1 ms ±  26.9 ms    [User: 248.4 ms, System: 107.8 ms]
  Range (min … max):   443.6 ms … 542.5 ms    10 runs

Benchmark 2: command ls -1 (brew --cellar)
  Time (mean ± σ):      10.0 ms ±   0.9 ms    [User: 3.4 ms, System: 4.3 ms]
  Range (min … max):     8.7 ms …  14.2 ms    112 runs

Summary
  'command ls -1 (brew --cellar)' ran
   47.16 ± 5.01 times faster than 'brew list --formula'
```

Reply:
@razvanazamfirei Is this a strategy that could be used in bash/zsh completions too? May want to split it out into a command like `brew formulae` if so.

Reply:
@razvanazamfirei Thanks! That could be a follow-up PR, doesn't need to block this one.